tox.ini

```diff
@@ -19,14 +19,15 @@ extras =
     develop
     test
 commands =
-    py.test --cov=src --cov-report=xml {posargs}
+    py.test --cov=src --cov-report=xml --html=report-{envname}.html --self-contained-html {posargs}
```

Comment:
Why do we need a n HTML report for each Python environment? What are you trying to achieve?